lib/core/check_for_lang_updates.js

```diff
@@ -43,7 +43,13 @@
   } else if (process.argv[2] === "add") {
     const type = process.argv[3];
     const name = process.argv[4];
-
+    let names = [];
+    let languages = 1;
+    if (name.includes(',')){
+      names = name.split(',');
+      names = names.filter(e => e);
+      languages = name.length
```

Comment:
this does not make sense to me, name is the argv passed to mc-build and a string.

Reply:
oops, I believe you're referring to `languages`. That's a typo

Reply:
assuming languages is meant to represent the number of languages being added im confused as to why its being set to the length of the parameter being passed.